solutions/example-ui-app/.eslintrc.js

```diff
@@ -2,7 +2,13 @@
 require('@rushstack/eslint-config/patch/modern-module-resolution');
 
 module.exports = {
-  extends: ['@aws/eslint-config-workbench-core-eslint-custom', 'next/core-web-vitals'],
+  extends: [
+    '@aws/eslint-config-workbench-core-eslint-custom',
+    'plugin:react/recommended',
+    'plugin:react-hooks/recommended',
+    'plugin:@next/next/recommended',
+    'plugin:react/jsx-runtime'
+  ],
```

Comment:
Ive seen these eslint plugins for React added in a couple packages, would it make sense to create a `@aws/eslint-config-workbench-core-eslint-custom-react` package to reduce the number of plugins React projects need to extend?

Reply:
@bhudnell yeah for sure, I will make the update in another PR